testsuite/python/lb_mass_conservation.py

```diff
@@ -41,7 +41,7 @@ class LBMassCommon:
 
     """Check the lattice-Boltzmann mass conservation."""
 
-    system = espressomd.System(box_l=[3.0, 3.0, 3.0])
+    system = espressomd.System(box_l=[6.0, 6.0, 6.0])
```

Comment:
Several mass conservation tests are know to take a lot of time, especially in code coverage builds. The runtime can significantly increase when multiple CI jobs run on the same runner, which is then starved of resources. This change may potentially increase the test runtime by a factor 8. Can you please confirm the runtime did not significantly change in the clang and coverage CI jobs, compared to the python branch?

Reply:
I changed the box_l from 6 to 4 to reduce the runtime. For testing with blocks_per_mpir_rank(i.e. [1,1,2]), box_l = 4 is needed.

Reply:
The total runtime for the clang CI jobs in this branch is 291.31 sec, compared to 271.33 sec in the python branch, which increases by around 7%.
The total runtime for the coverage CI jobs in this branch is 338.22 sec, compared to 320.22 sec in the python branch, which increases by around 6%.
These amounts cannot be ignored but are not so significant.